Automattic/wp-calypso
# Reader: Add load more button for Followed Sites

#### Changes proposed in this Pull Request

* POC for breaking the Followed Sites list into multiple pages for easier browsing.

I'm getting linting errors about using `setState` from within `componentDidUpdate` so this may not be a feasible approach. This is an older component so I didn't want to refactor it to use hooks, but maybe we should take the time to do that (or maybe there's a way to avoid using `setState` in `componentDidUpdate` here and I'm just not aware of it!)

Demo, obviously the button needs styling, but I didn't want to get too far down this road in case the above issue with setState is a blocker:

https://user-images.githubusercontent.com/2124984/135659257-37712919-5901-4766-ba0a-7770fa6e8956.mov


#### Testing instructions

* Switch to `activate/reader-sidebar`, then `add/load-more-button` (this PR builds on the former)
* Navigate to `/read` on a site with > 50 sites followed. 
* You should see a "Load more" button at the bottom of the Followed Sites list; when clicked, it should show the next set of 50 sites. Repeat until all sites have been shown.

Related to #56660 and #43027
